package.json

```
@@ -52,8 +52,8 @@
     "build:e2e:macos": "scripts/macos_e2e.sh 'build'",
     "bundle:android": "scripts/android_e2e.sh 'bundle'",
     "bundle:ios": "scripts/ios_e2e.sh 'bundle'",
-    "bundle:macos": "react-native bundle --entry-file index.js --platform macos --bundle-output example/index.macos.jsbundle --use-react-native-macos",
-    "test": "yarn test:lint && yarn test:flow",
+    "bundle:macos": "react-native bundle --entry-file index.ts --platform macos --bundle-output example/index.macos.jsbundle --use-react-native-macos",
+    "test": "yarn test:lint && yarn test:flow && tsc",
```

Can you add `test:ts` to run `tsc` to make it consistent? 

Also, what you think about using [concurrently](https://www.npmjs.com/package/concurrently) to run those in scripts concurrently? 

```suggestion
    "test": "concurrently -n 'lint,flow,ts' -c green 'yarn test:lint' 'yarn test:flow' 'yarn test:ts",
```